themes/default/content/blog/pulumi-watch-mode-with-rust/index.md

```diff
@@ -0,0 +1,150 @@
+---
+title: "Enabling Rapid Pulumi Prototyping with Rust"
+date: 2022-09-20T00:00:00-00:00
+
+meta_desc: >
+    Enabling Pulumi watch mode on Apple Silicon and solving Go cross-compilation by building on Rust
+meta_image: meta.png
+authors:
+    - aaron-friel
+tags:
+    - features
+    - watch
+    - rust
+---
+
+Pulumi enables engineers to employ the best practices of their field to infrastructure as code. The
+`pulumi watch` command is an example of this, enabling rapid prototyping and a "hot reload" style
+developer experience for prototyping Pulumi programs. In this post you'll see what watch mode
+enables, the challenges encountered in maintaining the feature, and how we were able to use Rust to
+bring that feature to more of our users.
+
+<!--more-->
+
+When developing a web application, hot reload frameworks enable quickly iterating and confirming the
+divs are centered. When implementing new features, a test runner with a watch mode makes test-driven
+development a breeze. In languages with an interactive shell, it's natural to use that to experiment
+with new APIs. In each of these cases, engineers find value in being able to focus on writing code,
+not typing console commands.
+
+That's why [watch mode exists]({{< relref "/blog/pulumi-watch-mode-fast-inner-loop-development-for-cloud-infrastructure" >}})
+and is one of my favorite features, and it's why I was disappointed to learn this feature [was not
+available to users on Apple Silicon Macs](https://github.com/pulumi/pulumi/pull/6497). As a Windows
```

Comment:
```suggestion
and is one of my favorite features, and it's why I was disappointed to learn this feature was not
available to users on Apple Silicon Macs([pr disabling watch mode](https://github.com/pulumi/pulumi/pull/6497)). As a Windows
```

Reply:
though im not sure the value in linking to this, to be honest.

Reply:
Let's nix it!